projects/scuba-physics/src/lib/blendPricing.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { BlendPricing, BlendCosts, BlendPrice } from './blendPricing';
+
+describe('BlendPricing', () => {
+    /** we will not test negative numbers */
```

Comment:
No longer true, since the validation is already added in the BlendPricing

Reply:
corrected

Reply:
added tests for negative numbers